src/main/java/edu/kit/ipd/dbis/database/connection/DatabaseManager.java

```diff
@@ -0,0 +1,243 @@
+package edu.kit.ipd.dbis.database.connection;
+
+import edu.kit.ipd.dbis.Filter.Filtersegment;
+import edu.kit.ipd.dbis.database.exceptions.sql.*;
+import edu.kit.ipd.dbis.org.jgrapht.additions.graph.PropertyGraph;
+
+import java.util.LinkedList;
+import java.util.Set;
+
+/**
+ *
+ */
+public interface DatabaseManager {
```

Comment:
Is this interface necessary?

Reply:
I don't think so. Leander asked this as well and I think we reached the conclusion that we should remove it.